packages/synapse-react-client/test/lib/containers/dataaccess/AccessHistoryDashboard.test.tsx

```diff
@@ -105,17 +116,23 @@ describe('AccessHistoryDashboard tests', () => {
     renderComponent()
 
     expect(await screen.findAllByRole('combobox')).toHaveLength(1)
+
+    expect(
+      screen.queryByTestId(SUBMISSION_TABLE_TEST_ID),
+    ).not.toBeInTheDocument()
+    expect(mockAccessRequestSubmissionTable).not.toHaveBeenCalled()
+    expect(screen.queryByTestId(APPROVAL_TABLE_TEST_ID)).not.toBeInTheDocument()
+    expect(mockAccessApprovalsTable).not.toHaveBeenCalled()
   })
 
-  // TODO: flaky in CI
-  it.skip('Renders table components and filter input for AR Name', async () => {
+  it('Renders table components and filter input for AR Name', async () => {
```

Comment:
Going to try to add this flaky test back in--hard to say if it's fixed since it really only breaks in CI, not locally. Perhaps changes in beforeAll/beforeEach could have made it more stable.